exonum-java-binding/common/src/test/java/com/exonum/binding/common/proofs/list/ListMerkleRootCalculatorTest.java

```diff
@@ -56,14 +60,16 @@ void visit_FullProof2elements() {
     ListProofElement right = leafOf(V2);
     ListProofBranch root = new ListProofBranch(left, right);
 
-    calculator = createListProofCalculator(root);
+    ListProof listProof = new ListProof(root, 2);
+    calculator = createListProofCalculator(listProof);
 
-    //calculate expected root hash
+    // Calculate expected merkle root
```

Comment:
Here and elsewhere, please use consistent terms: merkle root for the hash of the tree, and (? — don't remember) hash for the hash of the collection.